bcipy/helpers/system_utils.py

```diff
@@ -1,24 +1,46 @@
-import sys
+"""Utilities for system information and general functionality that may be
+shared across modules."""
+import importlib
+import logging
 import os
-
-from cpuinfo import get_cpu_info
-from typing import Optional, Tuple, Callable
+import pkgutil
+import platform
+import socket
+import sys
+import time
+import torch
 from pathlib import Path
+from typing import Callable, List, Optional, Tuple
+
 import pkg_resources
-import platform
 import psutil
 import pyglet
-import importlib
-import pkgutil
-import time
-import logging
+from cpuinfo import get_cpu_info
 
 DEFAULT_ENCODING = 'utf-8'
 DEFAULT_EXPERIMENT_ID = 'default'
 EXPERIMENT_FILENAME = 'experiments.json'
 FIELD_FILENAME = 'fields.json'
 DEFAULT_EXPERIMENT_PATH = '.bcipy/experiment/'
 DEFAULT_FIELD_PATH = '.bcipy/field/'
+REMOTE_SERVER = "https://github.com/CAMBI-tech/BciPy/"
+
+
+def is_connected(hostname: str = "one.one.one.one", port=80) -> bool:
```

Comment:
REMOTE_SERVER could be the default here

Reply:
Turns out this doesn't work if I'm on OHSU's VPN and it thought I was not on the internet. I updated this to use the "1.1.1.1" DNS resolver and it works both on and off VPN.